test/cli/_gen2/test_main.py

```diff
@@ -5,17 +5,21 @@
 import xarray as xr
 import yaml
 
+from test.s3test import MOTO_SERVER_ENDPOINT_URL
+from test.s3test import S3Test
+
 from xcube.cli._gen2.main import main
 from xcube.core.dsio import rimraf
 from xcube.core.new import new_cube
+from xcube.core.store import DataStorePool
 from xcube.core.store.stores.memory import MemoryDataStore
 
 
 class MainTest(unittest.TestCase):
     REQUEST = dict(input_config=dict(store_id='memory',
                                      data_id='S2L2A'),
                    cube_config=dict(variable_names=['B01', 'B02', 'B03'],
-                                    crs='WGS84',
+                                    crs='epsg:4326',
```

Comment:
What is the problem with "WGS84"?

Reply:
That's the problem with my pyproj installation. I revert this.